openhtf/io/user_input.py

```diff
@@ -157,7 +157,7 @@ def run(self):
         print self._message
 
         # Before reading, clear any lingering buffered terminal input.
-        termios.tcflush(sys.stdin, termios.TCIOFLUSH)
```

Comment:
This seems like an unrelated issue to the fix above, can you pull this out of this PR?